fortytools.yaml

```diff
@@ -106,7 +106,62 @@ paths:
           description: The app identified by the corresponding query parameter is not installed for any client of the user
         403:
           description: Invalid credentials, there is no User with this email / password combination.
-
+  /auth/firebase:
+    post:
+      tags:
+      - Auth
+      description: |
+        Performs a login to get firebase credentials
```

Comment:
Maybe we can align the description of this endpoint with the other one? as they are both doing the same thing; authentication; just with different input parameters.